src/Library/demos/Picture/main.blp

```diff
@@ -0,0 +1,49 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Picture");
+  description: _("A widget that displays images at their natural sizes");
+
+  child: Box {
+    orientation: vertical;
+
+    Picture picture {
+      halign:center;
+      valign:center;
+      can-shrink: bind can_shrink_switch.active;
+    }
+
+    Adw.Clamp {
+      maximum-size: 500;
+
+      Adw.PreferencesGroup {
+        title: _("Picture Properties");
+        margin-top : 12;
+
+        Adw.ComboRow content_fit_row {
+          title: _("Content Fit");
+          model: StringList {
+            strings ["Fill", "Contain", "Cover", "Scale Down"]
+          };
+        }
+
+        Adw.ActionRow {
+          title: _("Can Shrink");
+          activatable-widget: can_shrink_switch;
+
+          Switch can_shrink_switch {
+            valign: center;
+            active: true;
+          }
+        }
```

Comment:
I wonder if we shouldn't move these above the picture so they don't disappear when can shrink is set to false.